technic/spec/fixtures/default.lua

```diff
@@ -0,0 +1,15 @@
+
+-- Screwdriver is listed as optional but mod crashes without it
+_G.screwdriver = {}
```

Comment:
This won't be needed after #224 is merged, which should be merged before this PR, so that this can be removed.